README.md

```diff
@@ -37,6 +37,21 @@ If you encounter bugs or crashes, please [open a new issue](../../issues/new/cho
 
 To ask for help, join the [Discord](http://discord.gg/9hvwgeg).
 
+## Downloading
+
+1. Go to the [Actions](https://github.com/zeroeightysix/KAMI/actions) tab
+2. Click on the latest action
+3. Click on "KAMI mod jar"
+4. Extract downloaded zip file
+5. [Install](#installing)
```

Comment:
IIRC you need to be logged in to download an artifact. Mention this?

Reply:
I didn't know that. I'll fix it once I'm on a laptop